elements/rh-card/demo/inline-promo.html

```diff
@@ -0,0 +1,85 @@
+<rh-card variant="inline-promo">
+  <svg slot="inline-promo" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 190">
+    <title>A placeholder image in a card header slot</title>
+    <rect x="0" y="0" width="320" height="190" fill="#e7f0f9"/>
+    <line x1="0" y1="0" x2="320" y2="190" stroke="#d6e7f5"/>
+    <line x1="320" y1="0" x2="0" y2="190" stroke="#d6e7f5"/>
+  </svg>
+
+  <h2>Featured: left aligned with image</h2>
+  <p>You can customize the <code>#body</code> by using the <code>::part</code> selector. Using this technique, you can have padding on the top, bottom, left, right, or none at all.</p>
+  <rh-cta>
+    <a href="#">Call to action</a>
+  </rh-cta>
+</rh-card>
+
+<rh-card variant="inline-promo" reverse>
```

Comment:
do you think it's possible to avoid adding all these new boolean attributes like `reverse`, `one-col`, `full-width`, etc? each feature we add is a potential breaking change down the road when we inevitably discover that someone needs to modify or remove it

how much of this can be accomplished with existing tokens and parts?

Reply:
It's probably possible to avoid adding _most_ of them. I did use an adjacent sibling combinator for the `full-width` variant. Not sure if I can move that out of `rh-card.css` or not.

To be clear, do you want me to move any styles that can be accomplished via `::part()` to a `<style>` tag inside `inline-promo.html`? Then, if component authors would want a "one column" inline promo, they'd have to copy these styles to their pages as well.

Reply:
Following up here, I removed the `one-col`, `reverse`, `full-width`, and `standard` attributes/properties and converted them to classes on the `:host` in the demo (0309f97, decbb9b, f942a05).

I believe this is in line with what you're doing in #1590—minimizing our API surface and showing users it's possible to customize these components with their own CSS (via `::part` and styling the lightdom).

PTAL at [the DP](https://deploy-preview-1569--red-hat-design-system.netlify.app/elements/card/demo/inline-promo/) and let me know if I'm on the right track here. TIA!

Reply:
yeah looks good